stats/lib/platform-stats-fetchers.js

```diff
@@ -164,3 +164,20 @@ export const fetchParticipantsSummary = async (pgPool) => {
     participant_count: Number(rows[0].count)
   }
 }
+
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchDailyDesktopUsers = async (pgPool, filter) => {
+  const { rows } = await pgPool.query(`
+    SELECT
+      day::TEXT,
+      user_count::INT
```

Comment:
For my understanding, why are we casting to int here?

Reply:
In the query that was part of #327 I had to explicitly cast to int. I've just removed it here and it works fine. Thanks for raising this up 👍🏻